lib/ModelMixins/TableMixin.ts

```diff
@@ -44,6 +46,30 @@ import DiscretelyTimeVaryingMixin, {
 import ExportableMixin, { ExportData } from "./ExportableMixin";
 import { ImageryParts } from "./MappableMixin";
 
+class TableStratum extends LoadableStratum(TableTraits) {
+  static stratumName = "table";
+  constructor(private readonly _item: TableMixin.Instance) {
+    super();
+  }
+  duplicateLoadableStratum(newModel: BaseModel): this {
+    return new TableStratum(newModel as TableMixin.Instance) as this;
+  }
+
+  static load(item: TableMixin.Instance) {
+    return new TableStratum(item);
+  }
+
+  @computed
+  get disableOpacityControl() {
+    // disable opacity control for point tables - or if no mapItems
+    return (
+      this._item.activeTableStyle.isPoints() || this._item.mapItems.length === 0
+    );
+  }
```

Comment:
Would it be better to put this in `TableAutomaticStylesStratum`?

As it already has `disableDateTimeSelector` and a few similar things